x/examples/outline-connectivity/main.go

```diff
@@ -159,6 +210,28 @@ func main() {
 			if err != nil {
 				log.Fatalf("Failed to output JSON: %v", err)
 			}
+			// Send error report to collector if specified
+			if *reportToFlag != "" {
+				var samplingRate float64
+				if !success {
+					samplingRate = *reportFailureFlag
+				} else {
+					samplingRate = *reportSuccessFlag
+				}
```

Comment:
This is easier to reason about, so we don't need to think about a double-negative in the else case.
```suggestion
				if success {
					samplingRate = *reportSuccessFlag
				} else {
					samplingRate = *reportFailureFlag
				}
```